src/utils/signaling-connection.ts

```diff
@@ -0,0 +1,130 @@
+import * as map from "lib0/map";
```

Comment:
I was hoping that git would show this a file rename instead of an add and a delete. There is some minimum amount of changes to this file that is preventing that from happening.